open_spiel/python/examples/benchmark_games.py

```diff
@@ -57,8 +59,15 @@ def _rollout_until_timeout(game_name, time_limit, give_up_after):
       if len(state.history()) > give_up_after:
         num_giveups += 1
         break
-      action = random.choice(state.legal_actions(state.current_player()))
-      state.apply_action(action)
+      if not state.is_simultaneous_node():
+        action = random.choice(state.legal_actions(state.current_player()))
+        state.apply_action(action)
+      else:
+        actions = [
+          random.choice(state.legal_actions(i))
```

Comment:
Should we handle the case where there is no legal action? (Given https://github.com/deepmind/open_spiel/blob/e3a597b1d5e34a3130f36fa833702a1ae720a83f/open_spiel/spiel.h#L568)

Reply:
Done.